### PR TITLE
Grpc v2 fix trans bug

### PR DESCRIPTION
**Pull request by niki290595**

### CODE REVIEW

File: code/go/0chain.net/core/chain/entity.go [hunk 1]
```diff
@@ -21,8 +18,6 @@ func SetServerChain(c *Chain) {
 
 /*GetServerChain - returns the chain object for the server chain */
 func GetServerChain() *Chain {
-	logging.Logger.Info("call chain.GetServerChain",
```
Comment (kushthedude) [inline code/go/0chain.net/core/chain/entity.go:24]: Why did you remove this log?

Comment (niki290595) [reply code/go/0chain.net/core/chain/entity.go:24]: it clogs the log

**Review by kushthedude — APPROVED** [review]
